docs/DeveloperGuide.md

```diff
@@ -1,25 +1,88 @@
 # Developer Guide
 
-## Design & implementation
+## Design 
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Project overview
```

Comment:
Hi there! The project overview seems a little wordy. Might I suggest breaking it up to a few chunks ?
![image](https://user-images.githubusercontent.com/77775515/113082205-9ca11f80-920c-11eb-923f-f7f4af737d41.png)